UploadFile/Codeunit/Cod50410.BCPUploadFile.al

```diff
@@ -0,0 +1,57 @@
+codeunit 50410 "BCP UploadFile"
+{
+
+    procedure UploadFile(TableID: Integer; RecordNo: Code[20])
+
+    var
+        uploadTable: Record "BCP Upload";
+        FileInStream: InStream;
+        tempfilename: text;
```

Comment:
Just FileName, why temp?
also better to specify variables with capital letters

fileName
or
FileName

but not filename

Reply:
done